app/models.py

```diff
@@ -1581,6 +1593,10 @@ def mailbox_email(self):
         else:
             return self.user.email
 
+    def get_domain(self) -> str:
+        splitPos = self.email.find("@")
```

Comment:
nits: should we use underscore as variable name here?

Reply:
Maybe we can use this method `get_email_domain_part`

Reply:
Replace usages with `get_email_domain_part`